### PR TITLE
Document `RangeNode` fields

### PR DESCRIPTION
**Pull request by matthew-healy**
This PR adds documentation comments in `config.yml` for `RangeNode`'s fields.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -2317,12 +2317,40 @@ nodes:
...
+              1..3  # includes 3
+
+              1...3 # excludes 3
       - name: left
         type: node?
+        comment: |
+          The left-hand side of the range, if present. Can be either `nil` or
+          a node representing any kind of expression that returns a non-void
+          value.
+              1...
```
Comment (kddnewton) [inline config.yml:2332]:
```suggestion
          value.

              1...
```

**Review by kddnewton — CHANGES_REQUESTED** [review]
Looks so good! One minor change please!

**Review by kddnewton — APPROVED** [review]
Looks great

**Comment by matthew-healy** [comment]
Oops, somehow I missed the `comment` key. :see_no_evil: Will fix shortly.

**Pull request merged**